crates/egui/src/widgets/color_picker.rs

```diff
@@ -311,6 +335,32 @@ fn color_picker_hsvag_2d(ui: &mut Ui, hsva: &mut HsvaGamma, alpha: Alpha) {
     color_slider_2d(ui, v, s, |v, s| HsvaGamma { s, v, ..opaque }.into());
 }
 
+/// Shows 4 `DragValue` widgets to be used to edit the RGBA values.
+/// Alpha's `DragValue` is hidden when `Alpha::Opaque`.
+///
+/// Returns `true` on change.
+fn srgba_edit_ui(ui: &mut Ui, rgba: &mut [u8; 4], alpha: Alpha) -> bool {
+    let [r, g, b, a] = rgba;
+
+    let mut edited = false;
+    ui.horizontal(|ui| {
+        if ui.add(DragValue::new(r).speed(0.5).prefix("R: ")).changed() {
+            edited = true;
+        }
```

Comment:
Just realized that a `|` is also possible here. Also maybe using the `Widget`'s `ui` function might result in a nice `rustfmt` (since it's all function chains then)? So something like (but still needs a `fmt`): 

```suggestion
        edited |= DragValue::new(r).speed(0.5).prefix("R: ").ui(ui).changed();
```

Reply:
committed